src/test-utils/dom/anchor-navigation/index.ts

```diff
@@ -32,6 +32,18 @@ export default class AnchorNavigationWrapper extends ComponentWrapper {
   findAnchorLinkByHref(href: string): ElementWrapper<HTMLAnchorElement> | null {
     return this.find(`.${testUtilStyles['anchor-link']}[href="${href}"]`);
   }
+
+  /**
+   * Returns the wrapper of the first anchor that matches the specified test ID.
+   * Looks for the `data-testid` attribute that is assigned via `anchors` prop.
+   * If no matching anchor is found, returns `null`.
+   *
+   * @param {string} testId
+   * @returns {ElementWrapper<HTMLAnchorElement> | null}
+   */
+  findAnchorByTestId(testId: string): ElementWrapper<HTMLAnchorElement> | null {
```

Comment:
We expect anchor element as the return type, but it's assigned to `li`. Either return type or data-testid assignment should be adjusted

Reply:
Good callout, it actually needs to change a bit more to return the correct wrapper as well. 
To do that I added a new selector css class. It should be pointing to "Anchor Item" now rather than "Anchor Link" now. This is consistent with `findAnchorByIndex` and `findAnchors`.